src/app/phase-team-response/issues-responded/issues-responded.component.ts

```diff
@@ -61,8 +60,9 @@ export class IssuesRespondedComponent implements OnInit, OnChanges {
 
   ngOnInit() {
     this.filter = (issue: Issue) => {
-      return this.issueService.hasTeamResponse(issue.id) && !issue.duplicateOf &&
-        (issue.status === 'Done');
+      const isDuplicateIssue = (issue) => !!issue.duplicateOf;
+      const issueIsDone = (issue: Issue) => issue.status === STATUS.Done;
+      return this.issueService.hasTeamResponse(issue.id) && !isDuplicateIssue(issue) && issueIsDone(issue);
```

Comment:
I think it is more consistent to abstract out `this.issueService.hasTeamResponse` to a boolean value as well!